packages/remote-feature-flag-controller/src/utils/user-segmentation-utils.test.ts

```diff
@@ -1,14 +1,19 @@
+import { v4 as uuidV4 } from 'uuid';
+
 import {
   generateDeterministicRandomNumber,
   isFeatureFlagWithScopeValue,
 } from './user-segmentation-utils';
 
-const MOCK_METRICS_IDS = [
-  '123e4567-e89b-4456-a456-426614174000',
-  '987fcdeb-51a2-4c4b-9876-543210fedcba',
-  'a1b2c3d4-e5f6-4890-abcd-ef1234567890',
-  'f9e8d7c6-b5a4-4210-9876-543210fedcba',
-];
+const MOCK_METRICS_IDS = {
+  MOBILE_VALID: '123e4567-e89b-4456-a456-426614174000',
+  EXTENSION_VALID:
+    '0x86bacb9b2bf9a7e8d2b147eadb95ac9aaa26842327cd24afc8bd4b3c1d136420',
+  MOBILE_MIN: '00000000-0000-4000-8000-000000000000',
```

Comment:
Is this the minimum? I am wondering why the 8 is here
```suggestion
  MOBILE_MIN: '00000000-0000-4000-0000-000000000000',
```

Reply:
Nevermind, I verified experimentally by passing in minimum/maximum values to `uuid.v4()`'s `random` option, and we get exactly what you have here. I've also confirmed using `validate`. Interesting.